fetch_toolchain

```diff
@@ -7,16 +7,17 @@
 
 set -euo pipefail
 
-readonly PREFIX_JS=/tmp/toolchains/llvm-js
-readonly PREFIX_EMS=/tmp/toolchains/emscripten
-readonly tmpDir=/tmp/dscripten-tmp
-readonly scriptDir=$(realpath $(dirname $0))
+readonly PREFIX=${PREFIX:-/tmp/toolchains}
+readonly PREFIX_JS=$PREFIX/llvm-js
+readonly PREFIX_EMS=$PREFIX/emscripten
+readonly tmpDir=${tmpDir:-/tmp/dscripten-tmp}
+readonly scriptDir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)
```

Comment:
maybe it's simpler to use "readlink -f" here?

Reply:
Unless there is a need to actually resolve symlinks, it is generally best not to.